packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -45,26 +54,51 @@ const Dashboard = ({
         toggleStrategyModal();
     };
 
-    React.useEffect(() => {
-        if (active_tab === 0 && has_file_loaded) {
-            onEntered();
-        }
-        const tour_status = getTourSettings('status');
-
+    let tour_status: { [key: string]: string };
+    const setTourStatus = (param: { [key: string]: string }) => {
         if (tour_status) {
             const { action } = tour_status;
-            const actions = ['skip', 'close', 'reset'];
+            let actions = ['skip', 'close', 'reset'];
+
+            if (tourType.key === 'bot_builder_') {
```

Comment:
`tourType` 

 is this the correct convention to name an object? Can we double check in the guidelines?

Reply:
sure. checked. it should be tour_type